src/plugins/auto/src/plugin.cpp

```diff
@@ -287,11 +304,29 @@ ov::Any Plugin::get_property(const std::string& name, const ov::AnyMap& argument
     } else if (name == ov::device::full_name) {
         return decltype(ov::device::full_name)::value_type {get_device_name()};
     } else if (name == ov::device::capabilities.name()) {
-        auto device_list = get_core()->get_available_devices();
+        std::vector<std::string> device_list = arguments.count(ov::device::priorities.name())
+                                                   ? m_plugin_config.parse_priorities_devices(
+                                                         arguments.at(ov::device::priorities.name()).as<std::string>())
+                                                   : get_core()->get_available_devices();
+        bool enable_startup_cpu = arguments.count(ov::intel_auto::enable_startup_fallback.name())
+                                      ? arguments.at(ov::intel_auto::enable_startup_fallback.name()).as<bool>()
+                                      : true;
+        bool enable_runtime_cpu = arguments.count(ov::intel_auto::enable_runtime_fallback.name())
```

Comment:
if I remember right, runtime fallback does not have to be CPU always？ if AUTO:GPU,NPU it can also trigger fallback

Reply:
Yes, will update this logic soon.

Reply:
updated.